src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageDetailControlModel.cs

```diff
@@ -140,6 +140,23 @@ protected override Task CreateVersionsAsync(CancellationToken cancellationToken)
             var latestPrerelease = allVersionsAllowed.FirstOrDefault(v => v.version.IsPrerelease);
             var latestStableVersion = allVersionsAllowed.FirstOrDefault(v => !v.version.IsPrerelease);
 
+            if (_nugetProjects.Any())
+            {
+                if (_nugetProjects.First().ProjectStyle.Equals(ProjectModel.ProjectStyle.PackageReference) && installedDependency != null)
+                {
+                    VersionRange installedVersionRange = VersionRange.Parse(installedDependency.VersionRange.OriginalString, true);
+                    NuGetVersion bestVersion = installedVersionRange.FindBestMatch(allVersionsAllowed.Select(v => v.version));
+                    DisplayVersion displayVersion = new DisplayVersion(installedVersionRange, bestVersion, additionalInfo: string.Empty);
+                    _versions.Add(displayVersion);
+                }
+                else if (installedDependency != null)
+                {
+                    VersionRange installedVersionRange = VersionRange.Parse(installedDependency.VersionRange.OriginalString, false);
+                    DisplayVersion displayVersion = new DisplayVersion(installedVersionRange, additionalInfo: string.Empty);
+                    _versions.Add(displayVersion);
+                }
+            }
```

Comment:
``` suggestion
            if (_nugetProjects.Any() && installedDependency != null) 
             { 
                if (_nugetProjects.First().ProjectStyle.Equals(ProjectModel.ProjectStyle.PackageReference)) {  // code }
                else { //code }
             }

```

The idea is to avoid checking for `installedDependency != null` in both `if` and `else` blocks.